### ISSUE TITLE
Missing Feature : account creation email confirmation and validation

### ISSUE THREAD

**Issue by mzaradzki** [JulienPageaud/mobile-data-scoring-app#289]
Upon account creation there is no email confirmation being sent ; such should be mandatory to validate the account
Alternatively validation of the phone number should be required

**Comment by cdimitroulas** [comment]
Currently the e-mail field is not required so it makes no sens to require an email confirmation. Would you like me to make this mandatory?

**Comment by i-nomads** [comment]
I'd say phone validation the moment you sign in right? Like you type in the phone number, hit apply for a loan, it sends an SMS and you add the validation code at the same time as the password

**Comment by cdimitroulas** [comment]
that sounds like a good idea. @mzaradzki @JulienPageaud what do you think?

**Comment by mzaradzki** [comment]
User can create profile 2 different ways : 
a) through the apply for a loan
b) through the login icon that offer the signup option
Remark : The apply for a loan will need some changes as it assumes the user is new ; it should offer two choices for new and existing but I will raise that in another ticker

**Deferred  validation** :
I would prefer to have deferred validation (based on the apps I recently signup with) :
first user signup and enter profile details with his phone number and potentially an email.
The phone number and the email should have matching "is_verified" fields.
While the info is marked as is_verified=false the user is shown a button for "verify your phone number now" and "verify your email address now".

copy : @JulienPageaud

**Comment by cdimitroulas** [comment]
okay that can be achieved. If we go ahead with implementing phone verification then the Twilio account will need to be upgraded to a full paid account otherwise nobody will be able to verify their phone number!

**Comment by mzaradzki** [comment]
whose Twilio account is it right now ?
Twilio can accept prepaid fixed amount like 20gbp to do our tests ?

**Comment by cdimitroulas** [comment]
yes, I have created a stride Twilio account. I can give you the login details on hangouts
